src/inmanta/server/server.py

```diff
@@ -1906,6 +1902,9 @@ def safe_get(input, key, default):
             result = await self._run_compile_stage("Recompiling configuration model", cmd, project_dir, env=os.environ.copy())
 
             stages.append(result)
+        except CancelledError:
+            pass
```

Comment:
not raise? 

(i.e. this will allow later compiler stages to start)

Reply:
This task is backgrounded. If we raise it it will be catched and logged in handle_result of add_background_task

Reply:
perhaps add a comment to clarify that

Reply:
done